packages/react-components/src/util/getJudgements.ts

```diff
@@ -0,0 +1,64 @@
+// Copyright 2017-2021 @polkadot/react-components authors & contributors
+// SPDX-License-Identifier: Apache-2.0
+
+import { DeriveAccountRegistration } from '@polkadot/api-derive/types';
+import { DisplayedJudgement } from '@polkadot/react-components/types';
+import { RegistrationJudgement } from '@polkadot/types/interfaces';
+import { RegistrarIndex } from '@polkadot/types/interfaces/identity/types';
+
+function extractIndexes (registrars: RegistrationJudgement[]) {
+  return registrars.map((judgement) => judgement[0]);
+}
+
+export type SortedJudgements = ({ judgementName: DisplayedJudgement, registrarsIndexes: RegistrarIndex[] })[];
```

Comment:
maybe it could be moved to react-components/src/util/types?

Reply:
✅